src/test/java/Homework24.java

```diff
@@ -0,0 +1,2 @@
+public class Homework24 extends BaseTest{
+}
```

Comment:
Use in test methods, the following logical split:   GIVEN – WHEN - THEN

Please, add Assert

For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/928e1881-0c84-4d9a-8327-71e67acf912d)